apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/README.md

```diff
@@ -4,22 +4,29 @@ This project provides a Maven plugin that reads meta data in a YAML based DSL an
 
 - JPA entity classes
 - Data transformation classes to convert incoming message objects into entity objects.
+- Sample SQL for use in creating Flyway migrations.
+- `CsvWriter` classes for use with RIF pipeline.
```

Comment:
This reminds me...we should put a ticket in backlog to remove support for CsvWriter.